sdk/identity/azure-identity/test/perf/CMakeLists.txt

```diff
@@ -9,12 +9,12 @@ set(CMAKE_CXX_STANDARD_REQUIRED True)
 
 set(
   AZURE_IDENTITY_PERF_TEST_HEADER
-  inc/azure/identity/test/secret_credential.hpp
+  inc/azure/identity/test/secret_credential_test.hpp
```

Comment:
`client_secret_credential_perf_test.hpp`?
BTW, why for all perf tests we do have them under `inc/azure/.../test/`, wouldn't it be sufficient to put them under `inc` alone?